package.json

```diff
@@ -106,8 +106,8 @@
     "agentkeepalive": "^3.4.1",
     "chalk": "^1.0.0",
     "lodash": "2.4.2",
+    "lodash-2": "4.17.4",
```

Comment:
Sorry for intruding, but what is this lodash-2 package? With 1 version deployed a year ago and having two weekly downloads on npm, it looks quite fishy to me. I also cannot find any reference to lodash-2 in the official GitHub org. I wouldn't want that appearing in my code base...

Reply:
Oh, I understand, I thought that `lodash-2` was to pin a lodash 2.x version and the main `lodash` package would be upgraded to latest. I guess the the names and versions makes it confusing 😃. Anyways, great work!